ReText/webenginepreview.py

```diff
@@ -172,6 +175,10 @@ def __init__(self, tab,
         self.editBox.verticalScrollBar().valueChanged.connect(self.syncscroll.handleEditorScrolled)
         self.editBox.resized.connect(self._handleEditorResized)
 
+        # When preview is scrolled, update the editor scroll accordingly
+        if hasattr(webPage, 'scrollPositionChanged'):
```

Comment:
This signal was introduced in Qt 5.7, and we require Qt 6. Maybe the `if` is not needed?

And the same below on line 236.